Bottom: 7d9980e41fc84975900e2eb78268a2d4d4cea9b5
Top:    4228d15ee2484eed0be121e1158cadbe45754f59
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 16:20:31 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 4fd7de0..658ef83 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -580,9 +580,6 @@ static void handle_tx(struct vhost_net *net)
 			}
 			break;
 		}
-		/* On error, stop handling until the next kick. */
-		if (unlikely(err < 0))
-			break;
 
 		zcopy_used = zcopy && len >= VHOST_GOODCOPY_LEN
 				   && !vhost_exceeds_maxpend(net)
